WBooks/src/app/components/IconTab/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Image } from 'react-native';
+
+import { ROUTES } from '../../../constants/routes';
+
+import styles from './styles';
+import { icons, ACTIVE, PASSIVE } from './constants/iconsName';
+
+class IconTab extends Component {
+  getIconName = routeName => {
+    switch (routeName) {
```

Comment:
I recommend you avoid the use of `switch`, and use map objects approach, something like this...
```js
mapRouteToIcon = {
   [ROUTES.Library]: 'library',
   [ROUTES.Settings]: 'settings',
   ...
}
```
So, u can access to the icon doing `mapIconToRoute[ROUTES.Library]`